### PR TITLE
Migrate from JS to TS

### PR DESCRIPTION
**Pull request by antuking**

### CODE REVIEW

File: src/main/pages/NewProjectPage.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import ProjectsPage from './ProjectsPage';
```
Comment (binhle59) [inline src/main/pages/NewProjectPage.ts:1]: 'ProjectsPage' is declared but its value is never read

File: codecept.conf.js [hunk 2]
```diff
@@ -66,26 +61,26 @@ exports.config = {
...
-              },
-              check: (I) => {
-                I.seeInCurrentUrl('/dashboard');
-              },
+        dfQA: {
+          login: (I) => {
+            I.amOnPage("/");
+            I.setPopupCookie(true);
+            I.amOnPage("/");
+            I.waitInUrl('/dashboard', 10);
```
Comment (nhuthm-setel) [inline codecept.conf.js:76]: There are several parts using '..', but in here you are using "..". Can we make it as the common symbol @antuking

File: codecept.conf.js [hunk 3]
```diff
@@ -66,26 +61,26 @@ exports.config = {
...
+          },
+          check: (I) => {
+            I.seeInCurrentUrl('/dashboard');
```
Comment (nhuthm-setel) [inline codecept.conf.js:79]: Same in here

**Review by nhuthm-setel — APPROVED** [review]
LTGM!

**Pull request merged**